project/BuildHelper.scala

```diff
@@ -0,0 +1,48 @@
+import sbt.Keys._
+import sbt._
+import scalafix.sbt.ScalafixPlugin.autoImport._
+import wartremover.WartRemover.autoImport._
+
+object BuildHelper {
+  lazy val settingsCommon = Seq(
+    libraryDependencies ++= Seq(
+      compilerPlugin(Dependencies.kindProjector),
+      compilerPlugin(Dependencies.silencer),
+      compilerPlugin(scalafixSemanticdb), // for scalafix
+      Dependencies.silencerLib
+    ),
+    Compile / compile / wartremoverErrors ++= Warts.all filterNot Set(
+      Wart.Nothing, // keep, false positives all around
+      Wart.Overloading,
+      Wart.Any, // keep, false positives all around
+      Wart.Equals, // keep, easier that way
+      Wart.ToString, // keep, easier that way
+      Wart.Product, // keep, false positives all around
+      Wart.Serializable, // keep, false positives all around
+      Wart.DefaultArguments // for constructors for PureConfig
+    ),
+    ThisBuild / scalafixDependencies ++= Seq(
+      Dependencies.scalazzi // https://github.com/scalaz/scalazzi
+    ),
+    scalacOptions ++= Seq(
+      "-Yrangepos", // for scalafix. required by SemanticDB compiler plugin
+      "-Ywarn-unused", // for scalafix. not present in sbt-tpolecat for 2.13
+      "-P:silencer:checkUnused"
+    ),
+    scalacOptions --= {
+      if (!sys.env.contains("TRAVIS"))
```

Comment:
Please remove this, I don't think it's needed for now.

Reply:
I think this is a good idea to fail on warnings in CI
locally, you don't want to fail on warnings, because it break `scalafix` workflow unfortunatelly (see the comment in the file below)